src/target/riscv/riscv.c

```diff
@@ -619,8 +619,17 @@ static int find_first_trigger_by_id(struct target *target, int unique_id)
 	return -1;
 }
 
+static inline uint64_t get_least_significant_zero_index(riscv_reg_t tdata2)
```

Comment:
1. Since the definition is not in a header I believe `inline` to be redundant.
2. `unsigned int` seems more appropriate here as a return type:
https://github.com/riscv-collab/riscv-openocd/blob/c85d4e1858840fe90a2fa0c521528338952d49da/doc/manual/style.txt#L168-L169
3. IMHO, `tdata2` is better renamed to`reg` or some other generic name -- there is nothing specific to `tdata2` about this function.
4. If you like: `get_least_significant_zero_index` -> `count_trailing_ones` (seems to me more specific and closer to naming scheme of compiler builtins).
```suggestion
static unsigned int count_trailing_ones(riscv_reg_t reg)
```

Reply:
addressed